examples/medplum-hello-world/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "medplum-hello-world",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "@emotion/react": "11.10.8",
+    "@mantine/core": "6.0.9",
```

Comment:
These version numbers are out of sync w/ mantine in `app` and `react`.

Can we manually sync version numbers for now?  Or we can update the `scripts/upgrade.sh` tool to include `examples`, and do an automated update.

It's important that we keep version numbers in sync to maintain our sanity.